build/bin/user_setup

```diff
@@ -0,0 +1,13 @@
+#!/bin/sh
```

Comment:
This file isn't needed, and can be deleted.

Reply:
Done